src/providers/transcriptsProvider.ts

```diff
@@ -0,0 +1,17 @@
+import { HaDataProviderType } from './HaDataProviderType'
+import { transcriptApi } from './api'
+
+const transcriptsProvider: HaDataProviderType = {
+  async getList(page, perPage, filter) {
+    const result = await transcriptApi().getStudentTranscripts(filter.studentId, page, perPage)
+    return result.data
+  },
+
+  async getOne() {
+    //TODO: implements getOne provider. (tips) parse url
```

Comment:
__NB: yeah, but we may not want to parse any url
we'll talk about this one later too